README.md

```diff
@@ -1,24 +1,80 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-App
 
-Things you may want to cover:
+> This web application is a Ruby on Rails application that allows you to manage your budget in a professional way. You can register and/or log in and your data will be safely saved and you will be the only one person to manage access them. You can perform transaction transactions associated with a category of your transaction then at the end of the day you can see the money spent on each category.
 
-* Ruby version
 
-* System dependencies
+## Built With
 
-* Configuration
+- Major: Ruby, HTML,CSS
+- Frameworks: Rails, Bootstrap5
+- Libraries: Cancancan, Devise
+- Technologies: Git, Gitflow, Linters, Github, PostgreSql
 
-* Database creation
+## Live Demo (not available yet)
 
-* Database initialization
+[Live Demo Link](https://budgetify-ap.herokuapp.com/)
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+## Getting Started
 
-* Deployment instructions
 
-* ...
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Have a computer and internet connection
+- Have Ruby installed on your computer
+- Have Rails installed on your computer
+- Have a basic knowledge of Ruby and OOP concept
+- Have a basic knowledge of Rails
+- Have a general understanding of what testing is
+- Have visual-studio code or any other code editor installed on your computer.
+
+### Setup
+- Open your terminal in the folder where you want to have the project and run `git clone https://https://github.com/bruk19/Budget-App` to clone the project.
+- Run `cd Budget-App` to move to the project directory.
+
+### Install
+- Make sure to have installed POSTGRESQL
+
+- Run the command bellow in your terminal to get all required files
+
+   - Run `bundle install`  
+
+### Usage
+- Run `rails s` to start the app, then follow the given link provided in the console.
+- Run `rails c` to open an interactive ruby session
+
+### Run tests
+- Run `rspec`
+
+## Author
+
+👤 **Bruk Teshome**
+
+- GitHub: [@githubhandle](https://github.com/bruk19)
+- Twitter: [@twitterhandle](https://twitter.com/Bruktesh)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/bruk-teshome-ab4325226)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://https://github.com/bruk19/Budget-App/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
```

Comment:
* Kindly ensure your README contains an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). as stated in the [project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Original%20design%20idea,of%20your%20project.)